addons/resource.language.en_gb/resources/strings.po

```diff
@@ -23758,7 +23794,7 @@ msgstr ""
 #. Subtitle blur setting
 #: system/settings/settings.xml
 msgctxt "#39173"
-msgid "Blur"
+msgid "Border blur"
```

Comment:
this was right before, it is not related to the border

Reply:
My mistake, I only tested what it affected it with a border enabled and not without.